### ISSUE TITLE
CI check failing for https://github.com/pesto-students/rattle-battle/commit/b4ece6091625d970c433bdb2045329016876fb31

### ISSUE THREAD

**Issue by niinpatel** [pesto-students/rattle-battle#71]
This is a small issue really, but the CI check on for the commit [b4ece6091625d970c433bdb2045329016876fb31](https://github.com/pesto-students/rattle-battle/commit/b4ece6091625d970c433bdb2045329016876fb31) actually failed. 
But it is IMO a false negative, the result of the way we wrote the tests. 

The error message: 

`MongoError: E11000 duplicate key error collection: mockgoose-temp-db-0263553c-5328-4d74-a8d7-fb259c1ac8a0.leaderboards index: username_1 dup key: { username: "user6068" }
`

Basically, we're creating a users with username as random 4 digit numbers. So, if two usernames are same then mongo will throw an error. 

What should we do in this case, should we make a [revert commit](https://www.atlassian.com/git/tutorials/undoing-changes/git-revert)  and fix the issue, then remerge it? 

Should we let it go for now? And make a seperate PR for fixing the tests.

@nimish-gupta @vkbinfo @prakhar1912

**Comment by niinpatel** [comment]
do nothing.
